src/playlist-loader.js

```diff
@@ -146,28 +177,36 @@ export const isPlaylistUnchanged = (a, b) => a === b ||
   * master playlist with the updated media playlist merged in, or
   * null if the merge produced no change.
   */
-export const updateMaster = (master, media, unchangedCheck = isPlaylistUnchanged) => {
+export const updateMaster = (master, newMedia, unchangedCheck = isPlaylistUnchanged) => {
```

Comment:
I continuously got confused by `media` and `playlist` here and realized that they are just terrible names. Most of the changes in this function, other than those noted,  are for this rename.

Reply:
thought (non-blocking): yeah, I'm sure have a lot of places where the terminology is confusing. We probably want to write up some guidelines around naming, particularly around words like `media`, `playlist`, etc.